ninfod/ninfod.c

```diff
@@ -151,7 +151,10 @@ __inline__ static const char * log_level(int priority) {
 	}
 }
 
-void DEBUG(int pri, char *fmt, ...)
+#ifdef __GNUC__
+__attribute__((__format__ (__printf__, 2, 3)))
+#endif
```

Comment:
@kerolasa Could you please move this as a some definition into iputils_common.h (undef on `#else /* __GNUC__ */`) and use this definition, instead of copy paste whole thing?

Reply:
Okay: 59838bf4b39542abbdd93f58f046c8e9c2b1a43b